extensions/ConfirmEdit/ReCaptcha.php

```diff
@@ -43,23 +34,25 @@
 
```

Comment:
On line 31 there's this line ```$wgReCaptchaTheme = 'white';```. I don't think you need that, I can't find that variable being used anywhere.